.github/workflows/tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         with:
           submodules: true
       - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@d0e72ca3bfdc51937a4f81431ccbed269ef9f2a2
```

Comment:
There are more uses of `actions-rs/toolchain` in this workflow. Would it make sense to switch those also?